07_system_overview.R

```diff
@@ -978,9 +978,9 @@ inflow_outflow_df <- reactive({
 
       OutflowTypeDetail = case_when(
         perm_dest_client == TRUE ~
-          "Exited to \nPermanent Destination",
+          "Permanent\nDestination",
```

Comment:
I thought we decided to relabel this: "Exited, Permanent", to be consistent with the Sankey chart. Any reason you can think of NOT to do this?